tests/helm/testdata/goldenfile/logs_otc_daemonset/additional.output.yaml

```diff
@@ -95,148 +97,15 @@ spec:
               protocol: TCP
       initContainers: # ensure the host path is owned by the otel user group
         - name: changeowner
+          # yamllint disable-line rule:line-length
           image: public.ecr.aws/docker/library/busybox:1.36.0
           imagePullPolicy: IfNotPresent
           securityContext:
             capabilities:
-              drop:
-                - ALL
```

Comment:
Why does anything other than the namespace change in this file?

Reply:
@swiatekm-sumo I have reverted unwanted changes added by local output file regenerate. Didn't noticed these automatic updates from local make regenerate.